roles/setup/tasks/setup-CentOS.yml

```diff
@@ -21,6 +21,7 @@
     - python3-lxml
     - libvirt-client
     - virt-install
+    - cloud-init
```

Comment:
Installing `cloud-init` only to get `cloud-localds` is not a good idea. cloud-init contains much more than just `cloud-localds`, it does intrusive changes to the host. We want to keep the impact on the host minimal to not break the user.

Reply:
i see i see.
i think this should do it for me then:
```python
    cloud_localds_path =  module.run_command('which cloud-localds')
    if cloud_localds_path:
        cmd = cloud_localds_path
    else:
        alt_path = 'if [ -d "/usr/local/sbin/cloud-localds" ]; then  echo "true" fi'
        if alt_path == 'true':
            cmd = '/usr/local/sbin/cloud-localds'
     else:
       return module.fail_json(msg="could not find cloud-localds on your libvirt-host machine!")
```